.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
+*.xpi
+*~
```

Comment:
`server.js` might still be subjected to changes in future, but it would make sense to add that to `.jpmignore`(I've added the changes in the latest commit).